app/controllers/users_controller.rb

```diff
@@ -1,3 +1,26 @@
 class UsersController < ApplicationController
-  def new; end
+  def new
+    @user = User.new
+  end
+
+  def show
+    @user = User.find params[:id]
```

Comment:
Find tìm ko ra object user thì có sao ko em

Reply:
Dạ em fix ở pull trước rồi á chị, em có dùng rescue_from ActiveRecord::RecordNotFound để chuyển tới trang lỗi not found 404 ạ.